app/src/main/java/io/gnosis/safe/ui/dialogs/EnsInputViewModel.kt

```diff
@@ -18,7 +19,10 @@ class EnsInputViewModel
                 it ?: throw EnsResolutionError()
             }
             .onFailure {
-                throw EnsResolutionError(it.cause?.localizedMessage ?: it.localizedMessage)
+                if (it is Offline)
+                    throw it
```

Comment:
I really prefer braces on multiline if-else. But this is just a matter of taste and bad experience without blocks.